B/bitshuffle/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "bitshuffle"
+version = v"0.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiyo-masui/bitshuffle.git", "a60471d37a8cbbd8265dc8cfa83a9320abdcb590"),
+    GitSource("https://github.com/facebook/zstd.git", "e47e674cd09583ff0503f0f6defd6d23d8b718d3")
```

Comment:
Why adding this as a source?  It's already a dependency

Reply:
Added as a source to get the header file. Not sure how to get hold of that otherwise.

Reply:
`-I${includedir}`

Reply:
So the dependency `_jll`s actually contain header files? Impressive. I've updated the build script accordingly.